pkg/karmadactl/get.go

```diff
@@ -123,62 +119,64 @@ type CommandGetOptions struct {
 }
 
 // NewCommandGetOptions returns a GetOptions with default chunk size 500.
-func NewCommandGetOptions(parent string, streams genericclioptions.IOStreams) *CommandGetOptions {
-	return &CommandGetOptions{
-		PrintFlags: get.NewGetPrintFlags(),
-
-		CmdParent: parent,
-
-		IOStreams:   streams,
-		ChunkSize:   500,
-		ServerPrint: true,
-	}
-}
-
-// Complete takes the command arguments and infers any remaining options.
-func (g *CommandGetOptions) Complete(cmd *cobra.Command, args []string) error {
+func NewCommandGetOptions() *CommandGetOptions {
 	newScheme := gclient.NewSchema()
-	// human readable printers have special conversion rules, so we determine if we're using one.
-	g.IsHumanReadablePrinter = true
+	kubeConfig := defaultKubeConfig
+	printFlags := get.NewGetPrintFlags()
+
+	ioStreams := genericclioptions.IOStreams{In: getIn, Out: getOut, ErrOut: getErr}
 
```

Comment:
I suggest `NewCommandGetOptions` should be as brief as possible, more logic of initialization should be set in `Complete`.

This code logic comes from k8s, I advise not to change it.